specification/compute/resource-manager/Microsoft.Compute/stable/2020-12-01/compute.json

```diff
@@ -9273,6 +9276,11 @@
           "$ref": "#/definitions/SubResource",
           "description": "Specifies information about the proximity placement group that the virtual machine should be assigned to. <br><br>Minimum api-version: 2018-04-01."
         },
+        "platformFaultDomain": {
+          "type": "integer",
+          "format": "int32",
```

Comment:
is there any constraint on the int property?

Reply:
What do you mean by constraint?
There is a dynamic evaluation for it's max and min allowed values; with appropriate client errors.

Reply:
constaint means is there limitation on max/min value. So the comments means there are constraint right? If yes, better to add that in swagger to help user pass in right value

Reply:
No, I can't add it in swagger because the min -max values are dependent on the region, vmss, and other factors of how they created the vmss in first place. All that information is not appropriate to add in swagger.